nifi-registry-provider-api/src/main/java/org/apache/nifi/registry/hook/EventHookProvider.java

```diff
@@ -36,4 +36,18 @@
      */
     void handle(Event event) throws EventHookException;
 
+    /**
+     * Examines the values from the 'Event Whitelist X' properties in the hook provider definition to determine
+     * if the Event should be invoked for this particular EventType
```

Comment:
'Event Whitelist X' property has been renamed in this commit to 'Whitelisted Event Type '.